documentation/source/user_section/tutorials/shape-analysis.rst

```diff
@@ -12,12 +12,13 @@ This section demonstrates how to compute shape metrics of the spinal cord using
 .. toctree::
    :maxdepth: 1
 
-   shape-metric-computation/before-starting
-   shape-metric-computation/csa-averaged
-   shape-metric-computation/csa-perlevel
-   shape-metric-computation/csa-pmj
-   shape-metric-computation/csa-perslice
-   shape-metric-computation/other-shape-metrics
-   shape-metric-computation/verifying-correctness
+   shape-analysis/before-starting
+   shape-analysis/csa-averaged
+   shape-analysis/csa-perlevel
+   shape-analysis/csa-pmj
+   shape-analysis/csa-perslice
+   shape-analysis/other-shape-metrics
+   shape-analysis/verifying-correctness
```

Comment:
The tutorial sub-sections have a mix between verb forms (with and without 'ing'). I suggest using the shorter form (in this case: "verify correctness...").